homeassistant/helpers/template.py

```diff
@@ -1518,6 +1520,20 @@ def strptime(string, fmt, default=_SENTINEL):
         return default
 
 
+def default_na(value, default):
+    """Filter not assigned."""
+    if not value:
+        return default
+
+    if isinstance(value, jinja2.Undefined):
+        return default
+
+    if value in [STATE_UNKNOWN, STATE_UNAVAILABLE, ENTITY_MATCH_NONE]:
```

Comment:
I think this is incorrect. `ENTITY_MATCH_NONE` is not for these purposes and the state should not be able to be `None` either (as that is handled as unknown).

Reply:
I'll remove the None! Tks

Reply:
I'm open to ideas on new naming! 🤐

Reply:
Regarding naming, can you elaborate on the use case for this? I'm not a template expert so apologies if this is a question with an obvious answer.

I'm seeing two use cases:
1) undefined value
2) valid sensor value, but it has a potentially expected unavailable or unknown state e.g. due to a device being offline or whatever

Is it common to handle these the same way or is #1 a bug and #2 is expected to happen?

This seems kind of like `states('device_tracker.example', 'my_default_value')` so maybe including something about states in the name would make sense.